### ISSUE TITLE
[Bug] [1.20.1]Regions Unexplored nether biomes don't generate when bclib is the nether biome source

### ISSUE THREAD

**Issue by Questwizard** [quiqueck/BCLib#103]
### What happened?

When using the custom nether biome source in bclib (to get BetterNether biomes to properly generate), the nether biomes from Regions Unexplored don't generate at all. Regions Unexplored does use Terrablender, and from what they've said so far, they're adding and placing them normally in Terrablender. From what I can tell in the log, BCLib is picking up on the overworld biomes but not the nether ones.

If I disable the custom nether biome source, the Regions Unexplored nether biomes generate, but BetterNether biomes no longer generate fully.

### BCLib

3.0.8

### Fabric API

0.83.1

### Fabric Loader

0.14.21

### Minecraft

1.20.1

### Relevant log output

Full log file: [latest.log](https://gist.github.com/Questwizard/b0bdcac8d6aaff6bbe63a1f6570123f5)

### Other Mods

```shell
Full mod list is in the log above. Most likely relevant mods are the following:

Regions Unexplored 0.4.1+1.20.1
Terrablender 1.20.1-3.0.0.164
```

**Comment by quiqueck** [comment]
I think I already fixed this issue yesterday (c2a520e8460347edb0e879e0ddb022287ab0632d), so it should work in the next release.

However, it should already work if you change the WorldType back to the default one on the World Creation Screen. This will disables our custom BiomeSources, but we also register our End/Nether Biomes with fabric, so they should get picked up.

**Comment by quiqueck** [comment]
Should be fixed in BCLib 3.0.9. Please re-open the issue if you still experience Problems.
